README.md

```diff
@@ -1,6 +1,61 @@
-This is the Vagrant image used for running Cilium's integration tests.
-It contains a variety of dependencies Cilium needs, like Kubernetes, Docker,
-a Docker registry, etcd, and more.
+# cilium/packer-ci-build
 
-This was ported from https://github.com/eloycoto/cilium_basebox, which was
-created by @eloycoto as part of his work contributing to Cilium.
+This repo contains [packer](https://packer.io) templates and Jenkinsfiles to
+build the Vagrant boxes used for running Cilium's integration tests.
+
+The built Vagrant boxes contain a variety of Cilium build and test dependencies,
+like Go, Kubernetes, Docker, a Docker registry, etcd, and more.
+
+## Building Cilium Vagrant boxes locally
+
+1. Ensure you have [packer](https://www.packer.io/) installed.
+
+2. Make your changes locally and run
+
+   ```console
+   $ for d in ubuntu ubuntu-next ubuntu-4-19 ubuntu-5-4 ; do make DISTRIBUTION=$d ; done
+   ```
+
+   to build each of the four images.
+
+## Updating the Cilium Vagrant boxes used in tests
```

Comment:
Should we add a mention of the `build-*` triggers in here? Or directly reference https://docs.cilium.io/en/v1.10/contributing/testing/ci/#packer-ci-build, perhaps?

Reply:
I'd missed that section of the docs when writing this. I've replaced this section with a direct link.